lambda/objects/login.py

```diff
@@ -0,0 +1,4 @@
+class login(object):
+    def __init__(self, username = None, password = None, *args, **kwargs):
```

Comment:
cause as we make this as a SaaS in the future, it is better to make this an object if say that the username is REQUIRED then, it is easier for us to throw error quickly (just like in blog object) and this will be cleaner